src/getCloseTag.ts

```diff
@@ -0,0 +1,27 @@
+export default function getCloseTag(text: string, excludedTags: string[]): string {
+    const regex = /<(\/?[a-zA-Z][a-zA-Z0-9:\-_.]*)(?:\s+[^<>]*?[^\s/<>=]*)*\s?>/g;
```

Comment:
`'<abc =>` should not get close tag.

Reply:
The biggest concern is that: also this may improve the performance, but might break several kinds of scenario,

Reply:
`(?:\s+[^<>]*?[^\s/<>=]*)*` would make `[^\s/<>=]*` useless
`(?:\s+[^<>]*?[^\s/<>=]*)*` is similar to `(?:\s+[^<>]*?)*`

Reply:
Thanks for the advices  
what is `<abc =>` for? which languages please?

currently I hit again the very slow issue, I think I'll try a different approach than the big regex, and search backward with smaller regexes and a loop

Reply:
arrow function shouldn't trigger auto-close: https://github.com/formulahendry/vscode-auto-close-tag/issues/5

Reply:
search backward would be great idea! 👍

Reply:
oh you meant `'<abc =>'` like in a string? (or a comment, ..). Honestly I don't think this extension can be perfect, with jsx expressions we'd need to parse them to be 100% correct, and it's not even possible when we are in an unfinished one. So we might need to accept false-positiives like `<foo><bar x=1 y={'<lol'} ></`  that gives `lol` currently

Reply:
I don't mind if a few rare edge cases are broken, if that means the performance issues are fixed. Performance a big concern for this extension.

(I just wish we could add a timeout to regexps so they will fail if they take too long!)